tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include (LibAddTest)
 
 add_msr_test (kdb_get "${CMAKE_SOURCE_DIR}/doc/help/kdb-get.md REQUIRED_PLUGINS dump")
 add_msr_test (kdb_complete "${CMAKE_SOURCE_DIR}/doc/help/kdb-complete.md")
-# add_msr_test (kdb_cp "${CMAKE_SOURCE_DIR}/doc/help/kdb-cp.md")  # see #2600
+add_msr_test (kdb_cp "${CMAKE_SOURCE_DIR}/doc/help/kdb-cp.md")
```

Comment:
If you want to use `mini` in the MSR test for the file `kdb-cp.md`, then you should also add it as required plugin here. Otherwise the test will fail, if the current CMake configuration does not include the plugin. To do that, please replace

```cmake
add_msr_test (kdb_cp "${CMAKE_SOURCE_DIR}/doc/help/kdb-cp.md")
```

with

```cmake
add_msr_test (kdb_cp "${CMAKE_SOURCE_DIR}/doc/help/kdb-cp.md" REQUIRED_PLUGINS mini)
```

.

Reply:
Thanks!